Apps/Sandcastle/gallery/Atmosphere.html

```diff
@@ -154,6 +154,14 @@
     globe.lightingFadeInDistance = defaultLightFadeIn;
     globe.nightFadeOutDistance = defaultNightFadeOut;
     globe.nightFadeInDistance = defaultNightFadeIn;
+
+    document.getElementById('lightingFadeOutDistanceSlider').value = defaultLightFadeOut;
+    document.getElementById('lightingFadeOutDistanceText').value = defaultLightFadeOut;
+    document.getElementById('lightingFadeInDistanceSlider').value = defaultLightFadeIn;
+    document.getElementById('lightingFadeInDistanceText').value = defaultLightFadeIn;
+    document.getElementById('nightFadeOutDistanceSlider').value = defaultNightFadeOut;
+    document.getElementById('nightFadeOutDistanceText').value = defaultNightFadeOut;
+    document.getElementById('nightFadeInDistanceSlider').value = defaultNightFadeOut;
```

Comment:
I think the proper way to do this in knockout is:

```
viewModel.lightingFadeOutDistance = defaultLightFadeOut;
viewModel.lightingFadeInDistance = defaultLightFadeIn;
viewModel.nightFadeOutDistance = defaultNightFadeOut;
viewModel.nightFadeInDistance = defaultNightFadeIn;
```

Reply:
Thanks. Updated.